test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/HttpErrorTest.cs

```diff
@@ -72,6 +72,7 @@ public void ModelStateConstructorWithDetail_AddsCorrectDictionaryItems()
             modelState.AddModelError("[0].Name", "error2");
             modelState.AddModelError("[0].Address", "error");
             modelState.AddModelError("[2].Name", new Exception("OH NO"), metadata);
+            modelState.AddModelError("[2].Name", new Exception("OH NO 2"));
```

Comment:
Why change this test class? Eventual `ModelStateDictionary` additions for all non-`FormatException`s, non-`OverflowException`s, and non-`InputFormatterException`s are identical with and without the `ModelMetadata` argument.

Reply:
Okay. I just wanted to check that the overload without the metadata has an integration test of some sort. Suggestions for a better test class for that?

Reply:
I don't think an integration test or functional test covering these three added methods is necessary. The added unit tests are thorough enough.